src/activities/menu/Menu.qml

```diff
@@ -450,6 +498,106 @@ ActivityBase {
 
         }
 
+
+        Rectangle
+        {
+            id: searchbar
+            width:
+            {
+                if(horizontal)
+                    return parent.width/2
+                else
+                    return (parent.width -(section.width+10))
+            }
+
+            height:
+            {
+                if(horizontal)
+                    return parent.height/10
+                else
+                    return parent.height/17
+            }
+
+            visible:  false
+            anchors
+            {
+
+               bottom :
+               {
+                   if(horizontal)
+                       return activitiesGrid.top
+
+
+               }
+               top:
+               {
+                   if(!horizontal)
+                       return parent.top
+               }
+               left:
+               {
+                   if(!horizontal)
+                       return section.right
+               }
+
+
+
+
+            }
+            anchors.topMargin:
+            {
+                if(!horizontal)
+                    return 4
+            }
+            anchors.bottomMargin:
+            {
+                if(!horizontal)
+                    return 4
+            }
+
+
+            x: horizontal ? (parent.width/2) - (searchbar.width/2) : 0
+
+
+
+           opacity: 0.5
+           radius: 10
+           border.width: 2
+           border.color: "black"
+
+           gradient: Gradient {
+
+               GradientStop { position: 0.3; color: "#000" }
+               GradientStop { position: 0.9; color: "#666" }
+               GradientStop { position: 1.0; color: "#AAA" }
+
+           }
+
+            TextField
+            {
+                id : input
+                anchors.fill: parent
+                text: ""
+                textColor: "black"
+                font.pixelSize: 28
+                font.bold: true
+                font.family: gctext.fontfamily
```

Comment:
We should have a better way to access the font without using a unused text field. Maybe doing a GCTextField?